### PR TITLE
fix: updated readme

### LINKED ISSUE #71: Readme in root - howto use basic info

### PR DESCRIPTION
**Pull request by stevanlohja**
updated readme with simple usage. This closed #71

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+This repo contains a collection of example OpenRPC service definition files. These definitions can be found in the `service-desciptions/` directory:
+
+https://github.com/open-rpc/examples/tree/master/service-descriptions
+
+## Getting Started
+
+### Dependencies
+- Install the [Generator Client](https://github.com/open-rpc/generator-client)
+
+This is tool will build a client when passing a service definition - including _these_ examples.
```
Comment (shanejonas) [inline README.md]:
```suggestion
This is separate tool will build a client when passing a service definition - using the examples found in `service-descriptions/`.
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
 
 [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=open-rpc/examples)](https://dependabot.com)
 [![CircleCI](https://circleci.com/gh/open-rpc/examples.svg?style=svg)](https://circleci.com/gh/open-rpc/examples)
+
+This repo contains a collection of example OpenRPC service definition files. These definitions can be found in the `service-desciptions/` directory:
+
+https://github.com/open-rpc/examples/tree/master/service-descriptions
+
+## Getting Started
+
```
Comment (shanejonas) [inline README.md]: we can also link to examples via `playground`

https://playground.open-rpc.org?schemaUrl=https://raw.githubusercontent.com/open-rpc/examples/master/service-descriptions/simple-math-openrpc.json

Comment (stevanlohja) [reply README.md]: oh hell yeah.

**Comment by BelfordZ** [comment]
@stevanlohja any update on this?

**Pull request closed without merging**